test/signup.spec.js

```diff
@@ -28,16 +29,20 @@ describe('Signup Test', function() {
     });
 
     it('should verify user details', async function() {
-        await verifyUserOverApi();
+        await verifyUserByLogin(driver);
+        
+    });
+
+    it('should delete user account', async function() {
+        await navigateToProfile(driver);
+        await deleteUserViaUI(driver);
     });
 
     afterEach(async function () {
         await screenshot(driver, this.currentTest);
     });
 
     after(async () => {
-        // TODO: Test deletion via UI
-        await deleteUserOverApi();
```

Comment:
As already mentioned, I think it makes sense to make sure the user *really* got deleted / to really delete it in case of failure here.

Reply:
I updated it to use `verifyUserOverApi` if using UI didn't work.